docs/reference/platforms/linux/appimage.rst

```diff
@@ -41,6 +41,43 @@ AppImage:
    using Ubuntu 19.10, for example, you can expect that only people on the most
    recent versions of another distribution will be able to run your AppImage.
 
+Build Process
+=============
+
+Briefcase uses `linuxdeploy <https://github.com/linuxdeploy/linuxdeploy>`__ to
+build the AppImage in the correct format. AppImage tools expect directories in
+a specific format called the AppDir that will then be converted into the final
+AppImage. In practice, creating AppDirs for arbitrary applications tends to be
+a challenging task to get all of the dependencies properly bundled.
+
+Linuxdeploy simplifies this process as an AppDir maintenance tool while
+automating most of the process to create and bundle AppDirs for applications.
+It also includes a plugin system that makes bundling of frameworks easier.
+
+Building GTK Apps
+~~~~~~~~~~~~~~~~~
+
+In order to support GTK based apps like Toga in Linux, Briefcase also uses the
+`linuxdeploy-gtk-plugin
+<https://github.com/linuxdeploy/linuxdeploy-gtk-plugin>`__.
+This plugin goes through a series of steps to make sure that all the libraries
+are pulled in for the app to work properly once in the AppImage. The plugin
+goes through the following steps including:
+
+1.  Detects the GTK version to use, we manually set the version to 3
+2.  Installs itself as a hook in $APPDIR/apprun-hooks
+3.  Uses gsettings to set a light or dark adwaita theme
```

Comment:
I can see this is part of the plugin itself - but: what is the implication of this on the apps we distribution? Are all Briefcase GTK apps going to use the `adwaita` theme, no matter the user's local theme?

Reply:
Yes, that is a tradeoff of using AppImage. Themes have to be packaged with the app.